src/arrayMethodUnshift.js

```diff
@@ -5,7 +5,16 @@
  */
 function applyCustomUnshift() {
   [].__proto__.unshift2 = function(...elements) {
-    // write code here
+    const argumentsArrLength = elements.length;
```

Comment:
по сути, создание этой переменной только усложняет код :)
```suggestion
    const elementsLength = elements.length;
```